src/element-transformer/ElementTransformer.ts

```diff
@@ -3,9 +3,14 @@ import * as ts from 'typescript';
 
 require('ts-node').register();
 
+export interface CustomElementFile {
+	name?: string;
```

Comment:
If `name` represents the tag name, it might be a little more clear if we rename it to `tag`